### DISCUSSION TITLE
How to delay a transformation matrix

### DISCUSSION THREAD

**Discussion by Chryfi** [JacquesLucke/animation_nodes#1844] [Q&A] [answered]
I have one object that is parented to an armature and I want a second object to be parented with a delay to the first object so it follows it with a delay. I tried this but it doesn't work. If I move the first object, the second object just flies away
![Screenshot_318](https://user-images.githubusercontent.com/71967555/162968288-bdc58d7b-b505-4f66-89fc-ecb5dd8b7e2f.png)
.

**Comment by OmarEmaraDev** [comment, marked as answer]
The Transform Object node actually moves the object. Use the Object Matrix Output node instead.
Though I also don't see how this node tree will work. You should use a Delay Action instead. Take a look at this example:
https://docs.animation-nodes.com/release_notes/v2_1/#actions
